index.html

```diff
@@ -54,6 +57,28 @@ <h1 class="popheading">Tonic</h1>
                       </div>
             </div>    
            </div>
+           <div class="containerdesktop">
+            <div class="imgdivdesktop">
+                <img class="tproject" class="projects" src="image1desktop.png" alt="Nature">
+            </div>
+            <div class="tonicdesktop">
+                    <h1 class="popheading popheaddesktop">Tonic</h1>
+                    <p class="popp13 pop13desktop">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col pop13desktop">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col pop13desktop">2015</span></p>
+                    <br>
```

Comment:
- [ ] Please as stated by the previous 😃reviewer try to stick to the Figma Design, so please try to add some padding or margin between the left and right content of the section 😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/215074272-1fe1b82d-8e02-45ba-a24b-f09a4f80cff6.png) |    ![image](https://user-images.githubusercontent.com/83360636/215074188-1e315f65-a90c-48ae-9ed3-01e85e9741cb.png)|